spec/factories/answers.rb

```diff
@@ -1,6 +1,9 @@
 FactoryGirl.define do
   factory :answer do
-    body "This is the Answer body"
+    body 'This is the Answer body'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.